src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/model/PrisonerAlert.kt

```diff
@@ -1,10 +1,10 @@
 package uk.gov.justice.digital.hmpps.prisonersearch.model
 
-import io.swagger.annotations.ApiModelProperty
+import io.swagger.v3.oas.annotations.media.Schema
 
 data class PrisonerAlert (
-  @ApiModelProperty(required = true, value = "Alert Type", example = "H", position = 1)
+  @Schema(description = "Alert Type", example = "H")
```

Comment:
note that for kotlin don't need `required` any more, now based on whether the field is nullable